{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -35,35 +35,42 @@ def get_tm_session(session_factory, transaction_manager, request=None):
     - When using scripts you should wrap the session in a manager yourself.
       For example::
 
-          import transaction
+    .. code-block:: python
```

Comment:
For indentation, the initial `.` must align under the first letter of a list item, not the list item character and space (`- `).

```suggestion
      .. code-block:: python
```

Reply:
Also fix indentation in code below to be 4 spaces from under the first `.` in `.. code-block:: python`